client/my-sites/checkout/checkout/billing-details-form-fields.jsx

```diff
@@ -0,0 +1,591 @@
+					},
+					true
+				) }
+
+				{ this.createField( 'email', Input, {
+					label: translate( 'Email' ),
+				} ) }
+
+				{ this.createField( 'phone', FormPhoneMediaInput, {
+					label: translate( 'Phone' ),
+					onChange: this.handlePhoneChange,
+					countriesList: this.props.countriesList,
+					countryCode: this.state.phoneCountryCode,
+					enableStickyCountry: false,
+				} ) }
+
+				{ needsFax &&
+					this.createField( 'fax', Input, {
+						label: translate( 'Fax' ),
+					} ) }
+
+				{ this.createField(
+					'country-code',
+					CountrySelect,
+					{
+						label: translate( 'Country' ),
+						countriesList: this.props.countriesList,
+					},
+					true
+				) }
+
+				{ countryCode && (
+					<RegionAddressFieldsets
+						getFieldProps={ this.getFieldProps }
+						countryCode={ countryCode }
+						hasCountryStates={ hasCountryStates }
+						shouldAutoFocusAddressField={ this.shouldAutoFocusAddressField }
+					/>
+				) }
+			</div>
+		);
+	}
+
+	handleDomainDetailsChange = () => {
+		this.setState( { showDomainContactDetails: ! this.state.showDomainContactDetails } );
+	};
+
+	handleDomainPrivacyChange = event => {
+		if ( true === event.target.checked ) {
+			addPrivacyToAllDomains();
+		} else {
+			removePrivacyFromAllDomains();
+		}
+	};
+
+	getNumberOfDomainRegistrations() {
+		return cartItems.getDomainRegistrations( this.props.cart ).length;
+	}
+
+	needsDomainDetails() {
+		const cart = this.props.cart;
+
+		if ( cart && cartItems.hasOnlyRenewalItems( cart ) ) {
+			return false;
+		}
+
+		return (
+			cart &&
+			( cartItems.hasDomainRegistration( cart ) ||
+				cartItems.hasGoogleApps( cart ) ||
+				cartItems.hasTransferProduct( cart ) )
+		);
+	}
+
+	allDomainProductsSupportPrivacy() {
+		return cartItems.hasOnlyDomainProductsWithPrivacySupport( this.props.cart );
+	}
+
+	allDomainItemsHavePrivacy() {
+		return (
+			cartItems.getDomainRegistrationsWithoutPrivacy( this.props.cart ).length === 0 &&
+			cartItems.getDomainTransfersWithoutPrivacy( this.props.cart ).length === 0
+		);
+	}
+
+	renderDomainPrivacy() {
+		const { cart, translate } = this.props;
+		const renderPrivacy =
+			( cartItems.hasDomainRegistration( cart ) || cartItems.hasTransferProduct( cart ) ) &&
+			this.allDomainProductsSupportPrivacy();
+
+		if ( ! renderPrivacy ) {
+			return false;
+		}
+
+		return (
+			<label className="checkout__domain-privacy-label">
+				<FormCheckbox
+					checked={ this.allDomainItemsHavePrivacy() }
+					className="checkout__domain-privacy"
+					onChange={ this.handleDomainPrivacyChange.bind( this ) }
+				/>
+				<span>
+					{ translate( 'Register my domain with privacy protection (recommended)' ) }
+					<InfoPopover
+						className="checkout__item-tip-info"
+						position={ isMobile() ? 'left' : 'right' }
+					>
+						{ translate(
+							'Protects your identity and prevents spam by keeping your contact information off the internet.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Protects your identity and prevents spam by keeping your contact information off the Internet.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).